### ISSUE TITLE
Provide additional insights for the upcoming Glory changes

### ISSUE THREAD

**Issue by nathanhosselton** [Bungie-net/api#1033]
The [9/5/2019 TWAB](https://www.bungie.net/en/Explore/Detail/News/48124) went into some more details regarding the upcoming changes to Glory point reward/loss calculation, describing it as a function of the player's "expected" rank, itself based on the player's hidden skill rating.

I don't believe there is currently a way to see into a player's skill rating via the API, which has been fine since no one really needed to know that. But now that it's going to directly impact Glory progress, it would be nice if some part of this could be exposed for the sake of transparency in the progression system.

For instance, I've been working on [an app](https://github.com/nathanhosselton/Fabled) that calculates Glory trajectories for players and tells them how many wins they are away from certain ranks. I could do this because those win/loss values were static based on current rank.

Maybe the new system will be similar, with tiered static values based on a player's distance from their expected rank. This would make things super simple, as essentially all that would be needed (in my case) is for an endpoint (e.g `/Stats/Activities`) to be updated to include that distance value.

Bungie may understandably not want this, as it could indirectly expose players' "expected rank" and therefore their skill rating, which has historically been secret knowledge. My hope is that maybe Bungie will be okay with this now that skill rating is intended to eventually align with Glory rank.

On the flipside, it wasn't mentioned if the skill rating for other players within a match would affect the Glory reward/loss result for that match. If it will, then there is likely no real hope here, at least for my case.

But maybe something could still be opened up for others to leverage. I'm only making this request from my personal perspective, so you/others may have better ideas anyway.

**Comment by vthornheart-bng** [comment]
Sorry about not replying for so long - I missed the notification about this post, and going through the backlog I just spotted it!

Unfortunately, I don't have much I can convey.  All we've got is the data that comes down from the game content as far as being able to help with this info.  If there's anything hidden or secret contributing to it that's not in that data, BNet won't be able to get at it.  Sorry about that!
